backend/open_webui/retrieval/loaders/external_document.py

```
@@ -36,23 +36,48 @@ def load(self) -> list[Document]:
+        try:
+            headers["X-Filename"] = os.path.basename(self.file_path)
+        except:
+            pass
+
         url = self.url
         if url.endswith("/"):
             url = url[:-1]
 
-        r = requests.put(f"{url}/process", data=data, headers=headers)
+        try:
+            response = requests.put(f"{url}/process", data=data, headers=headers)
```

Ruff (0.11.9)

Probable use of `requests` call without timeout

(S113)